Traceback (most recent call last):
  File "/usr/local/lib/python3.10/dist-packages/jupyter_cache/executors/utils.py", line 58, in single_nb_execution
    executenb(
  File "/usr/local/lib/python3.10/dist-packages/nbclient/client.py", line 1256, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "/usr/local/lib/python3.10/dist-packages/nbclient/util.py", line 87, in wrapped
    return just_run(coro(*args, **kwargs))
  File "/usr/local/lib/python3.10/dist-packages/nbclient/util.py", line 62, in just_run
    return loop.run_until_complete(coro)
  File "/usr/lib/python3.10/asyncio/base_events.py", line 649, in run_until_complete
    return future.result()
  File "/usr/local/lib/python3.10/dist-packages/nbclient/client.py", line 694, in async_execute
    await self.async_execute_cell(
  File "/usr/local/lib/python3.10/dist-packages/nbclient/client.py", line 1012, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "/usr/local/lib/python3.10/dist-packages/nbclient/client.py", line 906, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
def my_graph_show(G):
    r"""Show a graph Greg's way.
    Prints the vertices and edges of the graph `G`.
    """
    print('vertices:',G.vertices(sort=True))
    print('edges:',G.edges(sort=True))
  
mydoc()
------------------

[0;31m---------------------------------------------------------------------------[0m
[0;31mTypeError[0m                                 Traceback (most recent call last)
Cell [0;32mIn [2], line 8[0m
[1;32m      5[0m     [38;5;28mprint[39m([38;5;124m'[39m[38;5;124mvertices:[39m[38;5;124m'[39m,G[38;5;241m.[39mvertices(sort[38;5;241m=[39m[38;5;28;01mTrue[39;00m))
[1;32m      6[0m     [38;5;28mprint[39m([38;5;124m'[39m[38;5;124medges:[39m[38;5;124m'[39m,G[38;5;241m.[39medges(sort[38;5;241m=[39m[38;5;28;01mTrue[39;00m))
[0;32m----> 8[0m [43mmydoc[49m[43m([49m[43m)[49m

[0;31mTypeError[0m: mydoc() missing 1 required positional argument: 'myfun'
TypeError: mydoc() missing 1 required positional argument: 'myfun'

